src/components/Dashboard/StaffNotation.vue

```diff
@@ -119,6 +119,10 @@ export default class StaffNotation extends Vue {
     font-size: 0.875rem;
     color: $app-blue;
     border-right: 1px solid $gray3;
+
+    &:hover {
```

Comment:
Just out of curiousity, what do you get if you don't specify the hover state?

Reply:
No difference in this case!

Reply:
Oh right, the hover state would be to change the look of this "button"... not used in this case.

Reply:
Totally, I guess i did that out of habit 😁